core/chains/evm/gas/chain_specific.go

```diff
@@ -28,5 +28,19 @@ func chainSpecificIsUsable(tx evmtypes.Transaction, cfg Config) bool {
 			return false
 		}
 	}
+	if cfg.ChainType() == config.ChainCelo {
```

Comment:
How do you ensure that this filtering logic doesn't filter out all the txs in this chain?
What if all txs are either of type 0x7c, or they're all having fees lower than BaseFeePerGas?